internal/status/templates/traces.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<!--
+ Copyright 2023 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>{{.Tool}} Dashboard</title>
+  <link href="/assets/main.css" rel="stylesheet" />
+  <!-- https://css-tricks.com/emoji-as-a-favicon/ -->
+  <link rel="icon" href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 viewBox=%220 0 100 100%22><text y=%22.9em%22 font-size=%2290%22>🧶</text></svg>">
+  <style>
+    /* Style for the trace table. */
+    #traces {
+      width: 100%;
+    }
+    #traces th, #traces td {
+      border: 1pt solid black;
+    }
+  </style>
+</head>
+
+<body>
+  <header class="navbar">
+    <a href="/">{{.Tool}} dashboard</a>
+  </header>
+
+  <div class="container">
+    <div class="card">
+      <div class="card-title">Traces</div>
+      <div class="card-body">
+        <table id="deployments" class="data-table">
+          <thead>
+            <tr>
+              <th scope="col">Trace URL</th>
+              <th scope="col">Start Time</th>
+              <th scope="col">Duration</th>
+            </tr>
+          </thead>
+          <tbody>
+            {{range .Traces}}
+              <tr>
+                <td><a href="{{traceurl .TraceID}}">link</a></td>
+                <td>{{.StartTime}}</a></td>
+                <td>{{dur .StartTime .EndTime}}</a></td>
```

Comment:
What is "dur"?  Would it be simpler to add a "Latency" field to the struct that is being rendered and just print that here?

Reply:
This is displaying the repeated version of an existing struct (i.e., `TraceSummary`), and I didn't want to invent another struct and convert into it.

I renamed "dur" -> "sub" to be more accurate.